source/lex.tex

```diff
@@ -673,13 +680,13 @@
 external source file names as specified in~\ref{cpp.include}.
 
 \pnum
-The appearance of either of the characters \tcode{'} or \tcode{\textbackslash} or of
+The appearance of either of the characters \unicode{0027}{apostrophe} or \unicode{005c}{reverse solidus} or of
 either of the character sequences \tcode{/*} or \tcode{//} in a
 \grammarterm{q-char-sequence} or an \grammarterm{h-char-sequence}
 is conditionally-supported with \impldef{meaning of \tcode{'}, \tcode{\textbackslash},
 \tcode{/*}, or \tcode{//} in a \grammarterm{q-char-sequence} or an
 \grammarterm{h-char-sequence}} semantics, as is the appearance of the character
-\tcode{"} in an \grammarterm{h-char-sequence}.
+\unicode{0022}{quotation mark} in an \grammarterm{h-char-sequence}.
```

Comment:
I don't think we want to use unicode notation for printable basic characters (that introduces inconsistencies even within that paragraph)

Reply:
I concur regarding this paragraph.  Please revert.

Reply:
Reverted and force-pushed.

Reply:
Note that this change seems to be an accidental repetition of the same change in #7336 which is marked as approved.  Should we revisit that PR too?

Reply:
On one hand, having lone special characters such as ' appear in text as (just) the glyph is a bit non-obvious, so the \unicode stuff helps for that; on the other hand, we don't make an attempt to present anything but glyphs for `//` or `/*`, which feels totally fine to me in isolation.

I'm not really sure which way we should go.  @tkoeppe ?

Reply:
As I have reverted that chance in this branch, and it is a duplicate of #7336, I suggest we have that conversation over there and keep the simplified PR here.

Alternatively, I could split #7336 in two so that this question has a PR all of its own.